lib/theme.js

```diff
@@ -1,7 +1,15 @@
-const colors = require('colors');
+/* eslint-disable no-console */
+const chalk = require('chalk');
 
-colors.setTheme({
-  info: 'green',
-  warn: 'yellow',
-  error: 'red',
-});
+exports.print = function print(msg, level) {
```

Comment:
also I think it's more clear to have level be the first parameter

Reply:
Discussed offline :)